src/Components/DataHistoryExplorerModal/DataHistoryExplorerModal.types.ts

```diff
@@ -1,12 +1,14 @@
 import { IStyle, IStyleFunctionOrObject, ITheme } from '@fluentui/react';
 import ADTDataHistoryAdapter from '../../Adapters/ADTDataHistoryAdapter';
 import MockAdapter from '../../Adapters/MockAdapter';
+import { IDataHistoryTimeSeriesTwin } from '../../Models/Constants/Interfaces';
 import { ICardboardModalStyles } from '../CardboardModal/CardboardModal.types';
 
 export interface IDataHistoryExplorerModalProps {
     adapter: ADTDataHistoryAdapter | MockAdapter;
     isOpen: boolean;
     onDismiss?: () => void;
+    timeSeriesTwins?: Array<IDataHistoryTimeSeriesTwin>;
```

Comment:
why is this optional?

Reply:
Because in one of your comments you asked for a story with the data already filled in (which might be also needed to feed in the modal with the selected twin&property in the property inspector anyhow), that is why I added this prop to open the data history explorer modal with a already added time series information.

Reply:
I see. Ok. If you think it actually makes sense to have besides just for the test that is fine. I just don't want us to be plumbing in props only for testing purposes.